test/integration/create/test_create.py

```diff
@@ -1055,7 +1055,48 @@ def select_data_func():
     assert can_select, "can not select data from the vhsard cluster"
 
 
-def check_create(tt_cmd, workdir, template, app_name, params, files):
+def get_status_info(tt_cmd, workdir, target):
+    status_cmd = [tt_cmd, "status", target]
+    p = subprocess.run(
+        status_cmd,
+        cwd=workdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True,
+    )
+    assert p.returncode == 0
+    return extract_status(p.stdout)
+
+
+def wait_cluster_started(timeout, tt_cmd, workdir, app_name, instances):
+    instances = [f"{app_name}:{inst}" for inst in instances]
+
+    def are_all_box_statuses_running():
+        status_info = get_status_info(tt_cmd, workdir, app_name)
+        print(f"wait_cluster_started: status_info: {status_info}")
+        running_instances = []
+        for key in status_info.keys():
+            if status_info[key].get("BOX") != "running":
+                return False
+            running_instances.append(key)
```

Comment:
```suggestion
        for inst, status in status_info.items():
            if status.get("BOX") != "running":
                return False
            running_instances.append(inst)
```
`RUNNING` - in upper case?

Reply:
Case is correct. This "running" doesn't refer to `STATUS` column of `tt status` output, it refers to `BOX` column that in turn comes from instance's `box.info.status`.